packages/airnode-examples/src/scripts/remove-airnode.ts

```diff
@@ -22,7 +22,7 @@ const main = async () => {
     integrationInfo.airnodeType === 'aws' && `--env-file ${secretsFilePath}`,
     integrationInfo.airnodeType === 'gcp' && `-v "${integrationPath}/gcp.json:/app/gcp.json"`,
     `-v ${integrationPath}:/app/output`,
```

Comment:
There's no need to mount the `output` directory but you need to mount the `config` directory instead. The same goes for the deployment.

Reply:
I made the changes in the docs but missed that here. Fixed, thanks.